form_app/lib/src/autofill.dart

```diff
@@ -31,13 +66,23 @@ class _AutofillDemoState extends State<AutofillDemo> {
                   ...[
                     Text('This sample demonstrates autofill. '),
                     TextFormField(
+                      autofocus: true,
+                      focusNode: _firstName,
+                      onFieldSubmitted: (value) {
+                        _lastName.requestFocus();
+                      },
                       decoration: InputDecoration(
                         hintText: 'Jane',
                         labelText: 'First Name',
                       ),
                       autofillHints: [AutofillHints.givenName],
                     ),
                     TextFormField(
+                      focusNode: _lastName,
+                      textInputAction: TextInputAction.next,
+                      onFieldSubmitted: (value) {
+                        _email.requestFocus();
```

Comment:
Manually focusing the next field is not required when using `TextInputAction.next`, so the FocusNodes can be removed.

Reply:
I didn't knew it, I've now removed unnecessary focusnodes